zanata-war/src/main/java/org/zanata/limits/RateLimitManager.java

```diff
@@ -125,7 +125,7 @@ public String getFieldValueAsString(String fieldName) {
                 });
     }
 
-    public RestCallLimiter getLimiter(final String apiKey) {
+    public RestCallLimiter getLimiter(final String key) {
```

Comment:
Can be username(if no api key generate with user), apiKey or anonymous IP address.